### ISSUE TITLE
Contact AutoCompletion not working

### ISSUE THREAD

**Issue by pierreozoux** [owncloud/mail#1345]
### Steps to reproduce
When I go to write a new email (or forward), I enter some letters in the To, CC, or BCC field, 

### Expected behaviour
There should be autocompletion.

### Actual behaviour
There are no autocompetion :/

**Mailserver or service:** (e.g. Outlook, Yahoo, Gmail, Exchange,...)
IndieHosters :)

**ownCloud Mail version build date:** (only if you are using a Nightly Build)
owncloud_mail_nightly_build_20160314.tar.gz

### Server configuration
**Operating system**:
debian jessie

**Web server:**
nginx

**Database:**
mysql

**PHP version:**
5.6

**ownCloud version:** (see ownCloud admin page)
OC9

**Updated from an older ownCloud or fresh install:**
Updated

**List of activated apps:**
Enabled:
  - activity: 2.2.1
  - calendar: 1.0
  - comments: 0.2
  - contacts: 1.0.0.0
  - dav: 0.1.5
  - documents: 0.12.0
  - federatedfilesharing: 0.1.0
  - federation: 0.0.4
  - files: 1.4.4
  - files_locking: true
  - files_pdfviewer: 0.8
  - files_sharing: 0.9.1
  - files_texteditor: 2.1
  - files_trashbin: 0.8.0
  - files_versions: 1.2.0
  - files_videoplayer: 0.9.8
  - files_videoviewer: 0.1.3
  - firstrunwizard: 1.1
  - gallery: 14.5.0
  - mail: 0.4.0
  - notes: 2.0.0
  - notifications: 0.2.3
  - provisioning_api: 0.4.1
  - systemtags: 0.2
  - tasks: 0.8.1
  - templateeditor: 0.1
  - updatenotification: 0.1.0
  - updater: 0.6
Disabled:
  - encryption
  - external
  - files_encryption
  - files_external
  - galleryplus
  - user_external
  - user_ldap
  - user_webdavauth

**The content of config/config.php:**
<?php
$CONFIG = array (
  'instanceid' => '',
  'passwordsalt' => '',
  'secret' => '',
  'trusted_domains' =>
  array (
    0 => 'cloud.pierre-o.fr',
  ),
  'datadirectory' => '/var/www/html/data',
  'overwrite.cli.url' => 'https://cloud.pierre-o.fr',
  'dbtype' => 'mysql',
  'version' => '9.0.0.19',
  'dbname' => 'owncloud',
  'dbhost' => 'db:3306',
  'dbtableprefix' => 'oc_',
  'dbuser' => 'root',
  'dbpassword' => '',
  'installed' => true,
  'mail_domain' => 'indie.host',
  'mail_from_address' => 'noreply.cloud.pierre-o.fr',
  'mail_smtpmode' => 'smtp',
  'mail_smtphost' => 'mail.indie.host',
  'mail_smtpport' => '587',
  'mail_smtpauth' => 1,
  'mail_smtpauthtype' => 'LOGIN',
  'mail_smtpname' => 'noreply.cloud.pierre-o.fr@indie.host',
  'mail_smtppassword' => '',
  'appstoreenabled' => true,
  'appstoreurl' => 'https://api.owncloud.com/v1',
  'theme' => '',
  'maintenance' => false,
  'memcache.local' => '\\OC\\Memcache\\APCu',
  'appstore.experimental.enabled' => true,
  'loglevel' => 0,
  'trashbin_retention_obligation' => 'auto',
);

```

### Client configuration
**Browser:**
ff dev edition and chromium

**Operating system:**
Ubuntu latest

### Logs
#### Web server error log
```
nothing relevant
```

#### ownCloud log (data/owncloud.log)
```
nothing relevant
```

#### Browser log
```
a) The javascript console log
no js logs :/

But I do get (when I navigate to contact app):
Refused to connect to 'http://remote.php/dav/addressbooks' because it violates the following Content Security Policy directive: "connect-src 'self'".

b) The network log 
nothing relevant
```

**Comment by DeepDiver1975** [comment]
What are you typing in and which name/mail address are you expecting this to match up with?

**Comment by pierreozoux** [comment]
So I have a contact that is:
First name: Pierre
Last Name: Ozoux
email: pierre@indie.host

And I type: 'pi' I expect to have this contact suggested.

Is it the expected behaviour?

On 16-03-2016 12:25, Thomas Müller wrote:
> What are you typing in and which name/mail address are you expecting this to match up with?
> 
> ---
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/owncloud/mail/issues/1345#issuecomment-197293486
> 

-- 
I use PGP to protect our privacy, if you want to know more, you can
follow this
https://emailselfdefense.fsf.org/en/

If you have further questions, please do not hesitate to ask.
You can verify my public key here: https://keybase.io/pierreozoux

**Comment by ChristophWurst** [comment]
@pierreozoux please test if #1394 fixes that bug for you :-)

**Comment by pierreozoux** [comment]
Yes, it does, thanks a lot!

Pierre

On 01-04-2016 08:49, Christoph Wurst wrote:
> Closed #1345 <https://github.com/owncloud/mail/issues/1345> via #1394
> <https://github.com/owncloud/mail/pull/1394>.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/owncloud/mail/issues/1345#event-610923777>
> 

-- 
I use PGP to protect our privacy, if you want to know more, you can
follow this
https://emailselfdefense.fsf.org/en/

If you have further questions, please do not hesitate to ask.
You can verify my public key here: https://keybase.io/pierreozoux
